### PR TITLE
Makes menubar always visible.

### PR DESCRIPTION
**Pull request by uzyn**
Not sure if it's just me. I did not know how to quit the app.

Thought of adding the quit button until I realized, upon looking at the source code, that the app is scrollable (and quit button is hiding underneath).

screen shot 2015-06-22 at 8 36 11 pm

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -40,7 +41,17 @@ a:hover, a:active {
   text-decoration: underline;
 }
 
+#results {
+  overflow: auto;
+  height: 430px;
```
Comment (cheeaun) [inline index.css]: Probably having a fixed height here is not a good idea, I think. Can try `height: calc(100% - 20px);`.

Comment (uzyn) [inline index.css]: :bow:

**Comment by uzyn** [comment]
Thanks. Made the change, 

Also made the menubar a little translucent to provide some indication that the main results area is scrollable.

screen shot 2015-06-23 at 9 24 03 am

**Comment by notthetup** [comment]
:+1:

**Comment by cheeaun** [comment]
:+1:

**Comment by sayanee** [comment]
e04c036ad0d382de89789213ec6b91a1965dcf80

**Pull request closed without merging**